modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/BaseSectionDisplayContext.java

```diff
@@ -828,6 +840,16 @@ private JSONObject _getJSONObject(long groupId) {
 		);
 	}
 
+	private String _getLayoutName() {
```

Comment:
@bcarvalho89, we add the layout name in several places.

@adolfopa, do you think it would be a good idea to have it as a protected method in the base class?

Reply:
I'd leave that out for the moment. Took a quick look and seems that most subclasses don't use that.